.github/workflows/build.yml

```diff
@@ -0,0 +1,57 @@
+name: Build
+on: [pull_request]
+
+jobs:
+  container-job:
+    env:
+      OED_DB_USER: test
```

Comment:
One nice benefit to a lot of these now is instead of having them in plaintext in the `.travis.yml` file, they can be put into the Github secrets section and then inserted like `${{ secrets.OED_DB_TEST_USER }}`.

Then perhaps down the road a test postgres db can be set up permanently that all the tests connect to.